Genome.xcodeproj/project.xcworkspace/xcshareddata/Genome.xcscmblueprint

```diff
@@ -0,0 +1,30 @@
+{
+  "DVTSourceControlWorkspaceBlueprintPrimaryRemoteRepositoryKey" : "EE0211841CDBC942EFB5905580DF4AA57B5992F6",
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyRepositoryLocationsKey" : {
+
+  },
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyStatesKey" : {
+    "EE0211841CDBC942EFB5905580DF4AA57B5992F6" : 0,
+    "3F07C23A561A9DDC76FA0BF032A7EA4B94C66928" : 0
+  },
+  "DVTSourceControlWorkspaceBlueprintIdentifierKey" : "488B3461-2C50-4DB1-BCE4-F2296D78A14D",
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyPathsKey" : {
+    "EE0211841CDBC942EFB5905580DF4AA57B5992F6" : "Genome\/",
+    "3F07C23A561A9DDC76FA0BF032A7EA4B94C66928" : "..\/.."
+  },
+  "DVTSourceControlWorkspaceBlueprintNameKey" : "Genome",
+  "DVTSourceControlWorkspaceBlueprintVersion" : 204,
+  "DVTSourceControlWorkspaceBlueprintRelativePathToProjectKey" : "Genome.xcodeproj",
+  "DVTSourceControlWorkspaceBlueprintRemoteRepositoriesKey" : [
+    {
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "github.com:djz-code\/CrossfaderTV.git",
```

Comment:
Is there a way we can remove this repository reference, or possibly the xcsmblueprint altogether?